charts/backstage/values.yaml

```diff
@@ -5,7 +5,7 @@ global:
     includes:
       # -- List of dynamic plugins included inside the `janus-idp/backstage-showcase` container image, some of which are disabled by default.
       # This file ONLY works with the `janus-idp/backstage-showcase` container image.
-      - 'dynamic-plugins.default.yaml'
+      - "dynamic-plugins.default.yaml"
```

Comment:
These should stay single quotes: https://support.asg.com/mob/mvw/10_0/mv_ag/using_quotes_with_yaml_special_characters.htm#:~:text=Single%20quotes%20let%20you%20put,returned%20as%20the%20string%20%5Cn.&text=Double%20quotes%20parse%20escape%20codes,as%20a%20line%20feed%20character.